### PR TITLE
chore: update adopters

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

As the title.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,22 +65,27 @@ This is a list of known adopters of Chaos Mesh. Some have already gone into prod
...
+- [KingNet](https://www.kingnet.com/)
+- [Meituan Dianping](https://about.meituan.com/en)
+- [PingCAP](https://en.pingcap.com/)
+- [Apache Pulsar](https://pulsar.apache.org/)
+- [Qihoo 360](https://360.cn/)
+- [Qiniu Cloud](https://qiniu.com/en)
+- [Tencent](https://www.tencent.com/en-us)
+- [Vald](https://vald.vdaas.org/)
+- [WeBank](https://www.webank.com/)
+- [Xpeng](https://en.xiaopeng.com/)
```
Comment (dcalvin) [inline README.md:77]: We should remove this list and add See Adopters, which links to the adopters.md file.

**Review by fewdan — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (g1eny0ung) [reply README.md:77]: 👌🏽，I will update later.

Comment (g1eny0ung) [reply README.md]: Done.

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Review by dcalvin — APPROVED** [review]
LGTM

**Pull request merged**